pkgs/by-name/ma/mattermost/package.nix

```diff
@@ -64,17 +106,20 @@ buildGoModule rec {
   ];
 
   postInstall = ''
-    tar --strip 1 --directory $out -xf $webapp \
-      mattermost/{client,i18n,fonts,templates,config}
+    mkdir -p $out/{client,i18n,fonts,templates,config}
+    cp -r ${webapp}/* $out/client/
+    cp -r ${src}/server/i18n/* $out/i18n/
+    cp -r ${src}/server/fonts/* $out/fonts/
+    cp -r ${src}/server/templates/* $out/templates/
+    OUTPUT_CONFIG=$out/config/config.json \
+      ${go}/bin/go run -tags production ./scripts/config_generator
 
     # For some reason a bunch of these files are executable
     find $out/{client,i18n,fonts,templates,config} -type f -exec chmod -x {} \;
```

Comment:
Is this still necessary? It was for a few old prebuilds but this may have fixed it.